tests/integration_tests/security/test_vulnerabilities.py

```diff
@@ -185,6 +185,33 @@ def spectre_meltdown_reported_vulnerablities(
     }
 
 
+def check_vulnerabilities_on_guest(status):
+    """
+    There is a REPTAR exception reported on m6i.metal for
+    guest kernel versions 4.14/5.10/6.1 when spectre-meltdown-checker.sh
```

Comment:
nit: As the failure does not depend on its kernel version, you might want to not mention guest kernel versions. Otherwise, that might make someone think that the exception doesn't happen on the other kernel versions.